Lib/timeit.py

```diff
@@ -243,6 +244,66 @@ def repeat(stmt="pass", setup="pass", timer=default_timer,
     return Timer(stmt, setup, timer, globals).repeat(repeat, number)
 
 
+def run(stmt="pass", setup="pass", timer=default_timer,
+        repeat=default_repeat, number=None,
+        time_unit=None, verbose=False):
+    precision = 3
+    if verbose:
+        precision += 1
```

Comment:
Also here:

```suggestion
    precision = 3 + verbose
```

Reply:
Hmm. So verbose=0 is precision 3 with no messages, verbose 1 is precision 4 with logs, verbose 2 is precision 5 with logs. Isn't that a bit weird?

Reply:
Is it clearer with `verbose_level`? It needs to be supported in the CLI.